sktime/forecasting/squaring_residuals.py

```diff
@@ -0,0 +1,399 @@
+# -*- coding: utf-8 -*-
+# copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
+"""Implements the probabilistic Squaring Residuals forecaster."""
+
+__all__ = ["SquaringResiduals"]
+__author__ = ["kcc-lion"]
+
+from warnings import warn
+
+import pandas as pd
+
+from sktime.datatypes._convert import convert_to
+from sktime.forecasting.base import BaseForecaster
+from sktime.forecasting.model_selection import ExpandingWindowSplitter
+from sktime.forecasting.naive import NaiveForecaster
+
+
+class SquaringResiduals(BaseForecaster):
+    r"""Compute the prediction variance based on a separate forecaster.
+
+    Wraps a `forecaster` with another `variance_forecaster` object that
+    allows for quantile and interval estimation by fitting the
+    `variance_forecaster` to the rolling residuals.
```

Comment:
I would use "volatility" or "residual", since variance has a specific meaning (of the residual distribrution) that does not apply for all choices of h

Reply:
or "scale", that´s another option